### ISSUE TITLE
ARIA attributes are used not according to spec

### ISSUE THREAD

**Issue by yatil** [swiegmann/kirby-menu#1]
In the example code in the README, it shows the following code:

```
<ul>
  <li>
    <a href="https://domain.com">Home</a>
  </li>
  <li aria-activedescendant>
    <a href="https://domain.com/news">News</a>
    <ul>
      <li aria-current>
        <a href="https://domain.com/news/technology">Technology</a>
      </li>
      <li id="meta-culture">
        <a href="https://domain.comn/news/culture">Culture</a>
      </li>
      <li id="meta-world" class="important" data-something>
        <a href="https://domain.com/news/world">World</a>
      </li>
    </ul>
  <li>
</ul>
```

`Aria-activedescendent` is for a secondary focus in a dropdown menu, it has no use in this context and might even break screen reader behavior. 

`Aria-current` is put on the `<li>` element, which is useless as the `<li>` is not focused. Furthermore, because the `aria-current` is not an HTML attribute, having no value makes no sense. (In contrast to [Boolean HTML attributes](https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML), having a valueless `aria-current` does not automatically default to `true`.) Put the `aria-current` attribute on the link directly and set it to `page`.

Similarly, if you want to signal what the current branch is (you currently try to use `aria-activedescendant` for that), use `aria-current` on the link (here to News) with a value of `location` or `true`.

Ref:

- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-activedescendant
- https://sarahmhigley.com/writing/activedescendant/

**Comment by swiegmann** [comment]
Thanks for noticing and clarifying. 👍 

I'll fix it in the next release.

**Comment by swiegmann** [comment]
Fixed: [Replaced wrong usage of ARIA-Attributes with CSS-Classes](https://github.com/swiegmann/kirby-menu/tree/39552b64ec2652cf4dbbfe71ef666aeead3f49c7)

**Comment by yatil** [comment]
👍🏼 The `aria-current=page` attribute should be set to the current link – otherwise the current page is only denoted using visuals, a violation of [WCAG SC 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships), and also confusing for some users.

**Comment by swiegmann** [comment]
Thank you,

I'll revisit that topic soon and re-introduce the `aria-current-page`-attribute then.

**Comment by yatil** [comment]
I think we’re talking across here. There is no attribute for you to introduce. You already detect which link is the current page. To that link, add in HTML `aria-current="page"` and you’re good. No need for users to configure anything :-)

**Comment by swiegmann** [comment]
Hey,

fixed it in [Added the ARIA-Attribute & value 'aria-current="page"' to the link-tag of the active page](https://github.com/swiegmann/kirby-menu/tree/5e0e47c690a2345bc11f6970b4b3ee94b6865250)

**Comment by swiegmann** [comment]
Hey @yatil,

I guess this can be closed, is this fixed for you?

Greetings

**Comment by yatil** [comment]
I think so :-)
